src/main/java/be/uantwerpen/minelabs/item/StarItem.java

```diff
@@ -0,0 +1,37 @@
+package be.uantwerpen.minelabs.item;
+
+import be.uantwerpen.minelabs.util.Tags;
+import net.minecraft.client.item.TooltipContext;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.ItemEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.text.Text;
+import net.minecraft.util.Hand;
+import net.minecraft.util.TypedActionResult;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class StarItem extends AbstractMagnet {
```

Comment:
Can we change the name to `BlackHoleItem` or something? I find this name confusing. Same for tags and other references.

Reply:
Is `POCKET_HOLE` fine ?

Reply:
I'm still afraid to Google it, but that probably says more about me than the name.

Reply:
![image](https://user-images.githubusercontent.com/7142721/234692357-289a9d36-970a-4d25-949c-8e399a58db65.png)